src/main/java/com/questhelper/helpers/quests/deathontheisle/DeathOnTheIsle.java

```diff
@@ -0,0 +1,734 @@
+
+		inspectWineLabels = new DetailedQuestStep(this, "Inspect the Wine labels in your inventory", wineLabels.highlighted());
+		inspectThreateningNote = new DetailedQuestStep(this, "Inspect the Threatening note in your inventory", threateningNote.highlighted());
+		inspectDrinkingFlask = new DetailedQuestStep(this, "Inspect the Drinking flask in your inventory", drinkingFlask.highlighted());
+		inspectShippingContract = new DetailedQuestStep(this, "Inspect the Shipping contract in your inventory", shippingContract.highlighted());
+
+		returnToTheGuards2 = returnToTheGuards.copy();
+
+		// varbit 11233: 0 -> 1 when handing over clues to the guards
+
+		investigateMurder2 = new ConditionalStep(this, returnToTheGuards2);
+		investigateMurder2.addStep(not(inVilla), returnToButlerAndHeadInside);
+		investigateMurder2.addStep(and(pickpocketedItems, not(inspectedWineLabels)), inspectWineLabels);
+		investigateMurder2.addStep(and(pickpocketedItems, not(inspectedThreateningNote)), inspectThreateningNote);
+		investigateMurder2.addStep(and(pickpocketedItems, not(inspectedDrinkingFlask)), inspectDrinkingFlask);
+		investigateMurder2.addStep(and(pickpocketedItems, not(inspectedShippingContract)), inspectShippingContract);
+		investigateMurder2.addStep(not(pickpocketedAdala), pickpocketAdala);
+		investigateMurder2.addStep(not(pickpocketedPavo), pickpocketPavo);
+		investigateMurder2.addStep(not(pickpocketedCozyac), pickpocketCozyac);
+		investigateMurder2.addStep(not(pickpocketedXocotla), pickpocketXocotla);
+
+		/// 27
+		talkToGuardsAgainToTellThemYouAreReady = new NpcStep(this, NpcID.STRADIUS, new WorldPoint(1433, 2935, 0), "Talk to the guards to tell them you're ready to make an accusation.");
+		talkToGuardsAgainToTellThemYouAreReady.addDialogStep("I'm ready.");
+		talkToGuardsAgainToTellThemYouAreReadyStep = new ConditionalStep(this, talkToGuardsAgainToTellThemYouAreReady);
+		talkToGuardsAgainToTellThemYouAreReadyStep.addStep(not(inVilla), returnToButlerAndHeadInside);
+
+		/// 28 + 30
+		// TODO: what happens if you accuse the wrong person?
+		interrogateConstantiniusAgain = new NpcStep(this, NpcID.CONSTANTINIUS, new WorldPoint(1446, 2931, 2), "Interrogate Constantinius.");
+		interrogateConstantiniusAgain.addDialogStep("No.");
+		interrogateXocotlaAgain = new NpcStep(this, NpcID.XOCOTLA, new WorldPoint(1444, 2928, 2), "Interrogate Xocotla.");
+		interrogateXocotlaAgain.addDialogStep("No.");
+		interrogateCozyacAgain = new NpcStep(this, NpcID.COZYAC, new WorldPoint(1442, 2929, 2), "Interrogate Cozyac.");
+		interrogateCozyacAgain.addDialogStep("No.");
+		interrogatePavoAgain = new NpcStep(this, NpcID.PAVO, new WorldPoint(1442, 2931, 2), "Interrogate Pavo.");
+		interrogatePavoAgain.addDialogStep("No.");
+
+		accuseAdala = new NpcStep(this, NpcID.ADALA, new WorldPoint(1446, 2933, 2), "Accuse Adala of the crime, ready for an easy fight. You can use Protect from Melee to negate all damage.");
```

Comment:
Fun thing I found, if you hit 1hp you say you yield and the quest continues, so seems safe.

Reply:
Removed any mention of protect from melee in [`900ec8b` (#1774)](https://github.com/Zoinkwiz/quest-helper/pull/1774/commits/900ec8b749f35206cf1ccc3b4b53fd58cf7efc39)